cypress/integration/create.js

```diff
@@ -0,0 +1,29 @@
+describe('Sourcebooks login', function() {
+
+    it('Admin should create a new client', function () {
+
+        const organizationName = Math.ceil(Math.random()*100000);
+
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Saulė Raudytė"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="Admin"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.get('[href="/clients"]').click();
+        cy.get('.btn').click();
+
+        cy.get('[id="clientDetailsForm.organization"]').type(organizationName);
+        cy.get('[id="clientDetailsForm.contacts_firstName_0"]').type(organizationName+1);
+        cy.get('[id="clientDetailsForm.contacts_lastName_0"]').type(organizationName+2);
+        cy.get('[id="clientDetailsForm.contacts_email_0"]').type(organizationName+'@email.com');
+        cy.get('[type="submit"]').click();
+        cy.wait(1000);
```

Comment:
this wait does not guarantee that test will always pass, what if for test to pass we needed to wait for 1.5s? I would suggest that after submit you check if success pop up was displayed at the top of page and then proceed.